src/lib/coaler/singlealign/SingleAligner.cpp

```diff
@@ -79,8 +79,8 @@ namespace coaler {
 
         if (match_vect_a.size() != core_structure->getNumAtoms()
             || match_vect_b.size() != core_structure->getNumAtoms()) {
-            spdlog::error("Core is not a common core structure of molecule a and molecule b!");
-            throw std::runtime_error("Core is not a common core structure of molecule a and molecule b!");
+            spdlog::error("Matcher is not a common core structure of molecule a and molecule b!");
```

Comment:
"Matcher is not a common core struct" klingt ein wenig weird

Reply:
fixed